Instant-Messenger/include/server/server_group_manager.h

```diff
@@ -0,0 +1,23 @@
+#include <string>
```

Comment:
Should not this be a .hpp file?

Reply:
yeap, our include files should be hpp, I'll changes this